apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -334,6 +340,45 @@ create_contract_init_error(_Cfg) ->
                  aec_accounts:balance(aect_test_utils:get_account(PubKey, S2))),
     ok.
 
+create_contract_init_error_the_invalid_instruction(_Cfg) ->
+    create_contract_init_error_illegal_instruction_(16#fe, <<"">>).
```

Comment:
Could note that `0xfe` is the instruction `INVALID`, and that `0x0c` and `0xfc` are "missing" op codes.

Reply:
I will address in new version of PR 1810.